caikit/runtime/http_server.py

```diff
@@ -544,6 +547,7 @@ def _dataobject_to_pydantic(
     @staticmethod
     def _health_check():
         log.debug4("Server healthy")
+        return "OK"
```

Comment:
can you get Micah to put a return type annotation here?

Reply:
The sharp eyed reviewer! He asked me to tell him what my co-workers said, so I'll report back to him

Reply:
🤣